docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,100 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` procedure prepares an Iceberg table for moving or copying to another location.
+
+### `rewrite-table-path`
+
+Stages a copy of the Iceberg table's metadata files where every absolute path source prefix is replaced by the specified target prefix.  
+This can be the starting point to fully or incrementally copy an Iceberg table located under an absolute path under a 
+source prefix to another under the target prefix.
```

Comment:
"under an absolute path under a source prefix to another under the target prefix."  is not easy to grok.

Reply:
Now  'another' seems like 'another Iceberg table'.  How about, just:

```
This can be the starting point to fully or incrementally move or copy an Iceberg table to a new location.
```
?